diff -up byacc-20070509/skeleton.c\~ byacc-20070509/skeleton.c
--- byacc-20070509/skeleton.c~	2011-10-05 15:30:19.673606102 +0200
+++ byacc-20070509/skeleton.c	2011-10-05 15:44:09.054625022 +0200
@@ -74,8 +74,8 @@ char *header[] =
     "#ifdef YYMAXDEPTH",
     "#define YYSTACKSIZE YYMAXDEPTH",
     "#else",
-    "#define YYSTACKSIZE 500",
-    "#define YYMAXDEPTH  500",
+    "#define YYSTACKSIZE 10000",
+    "#define YYMAXDEPTH  10000",
     "#endif",
     "#endif",
     "",

Diff finished.  Wed Oct  5 15:44:14 2011
